### ISSUE TITLE
First Learning Challenge (Update)

### ISSUE THREAD

**Issue by doubleshow** [bigdatahci2015/forum#3]
Please pay attention to the following updates: 
## Scope (NEW)
1. Team Component: Choose one of the two: SVG Template or Lodash
2. Individual Component: The other one not chosen for the team component

In other words, there are no more questions to solve other than those already in your book template. This way, you and your team can have more time to become familiar the structure of the course.
## Due
1. Team Component: Friday August 28, 11:59pm.
2. Individual Component: Sunday August 30, 11:59pm.
## Submission
1. Team Component: Everyone should make his or her own commit (which is automatically timestamped by Github)
2. Individual Component: Everyone should make his or her own commit (which is automatically timestamped by Github)

In other words, the submission methods are exactly the same for both team and individual components. You just need to do it two times each week, one by Friday night, then by Sunday night.
## Grading

This week, as we are still getting familiar with the system, you will make your best efforts to submit but do not need to worry about doing something wrong in submitting your homework. We will give you multiple chances to submit until everyone got it right.

If you have any question, simply post a rely to this issue.

**Comment by drewdinger** [comment]
Will we be submitting one submission for the team exercise or should all group members submit the team portion?

**Comment by doubleshow** [comment]
Everyone is responsible for submitting his/her own version of the solution. The major difference is that for the team portion, you can share the solution (copy and paste), but for the individual portion, you need to have your own solution.

**Comment by calebhsu** [comment]
Just a heads up that part of my group completed the Team challenge before it was split into Team & Individual components, so it might show as one commit on GitHub.

**Comment by karisantos** [comment]
Still a little unclear on submission.  Do I just commit to my fork?
For me: commit to karisantos/book  ?

**Comment by doubleshow** [comment]
@karisantos Yes. It will then be in your commit history that we can track and check.

**Comment by sumi6109** [comment]
When I am publishing my gitbook related to lodash ,I am getting this error.
unknown block tag: lodash (In file 'section0.md')

Any Idea?I have install lodash as global also 
npm install lodash -g
